gamemodes/horde/gamemode/sv_playerlifecycle.lua

```diff
@@ -285,6 +301,11 @@ function HORDE:GameEnd(status)
 
     net.WriteUInt(total_damage, 32)
 
+    net.WriteEntity(most_revives_player)
+    net.WriteUInt(most_revives, 32)
```

Comment:
Shouldn't be 32, you're never gonna have `4,294,967,295` revives. Something like `9` would be much better